sockeye/coverage.py

```diff
@@ -77,8 +81,8 @@ class CountCoverage(Coverage):
     Coverage class that accumulates the attention weights for each source word.
     """
 
-    def __init__(self, prefix='') -> None:
-        self.prefix = prefix
+    def __init__(self) -> None:
+        super().__init__()
```

Comment:
They all default to the default prefix for now which is not really a problem because loading the correct type of attention is handled via attention-type not the names of symbols in the graph.

If you want I can expose, but I don't see a good reason to name them differently right now.